LICENSE

```diff
@@ -25,3 +25,13 @@ INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
 CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 POSSIBILITY OF SUCH DAMAGE.
+
+--------------------------------------------------------------------------------
+
+GlobalPlatform API license
+
+Since OP-TEE is a TEE implementing GlobalPlatform APIs, users of OP-TEE should
+get the license for the GlobalPlatform TEE API either by joining GlobalPlatform
+as a member or, they can get it for free by going to and download the
```

Comment:
This is a mine field see for instance https://www.theregister.co.uk/2017/05/25/your_roadmap_to_the_google_vs_oracle_java/

At the end of the first paragraph in the GP license it says:
>For the avoidance of doubt, products implementing this Specification are not deemed to be derivative works of the Specification.

The text we're adding here is much more strict and makes in my mind this software less useful.